beRajeevKumar/Frontend_Mentor
# 3D planet Visualizer Added

What does this PR do?

I have integrated my mini 3D planet visualizer into the list of Frontend projects, enhancing the repository with an interactive 3D experience.

Fixes # (issue)


![Screenshot 2024-09-23 215412](https://github.com/user-attachments/assets/321a50f3-6194-44f7-bcf2-b7153c5e1656)
![Screenshot 2024-09-23 215456](https://github.com/user-attachments/assets/1f4a28c7-3ed6-4989-8653-7f0fe7cc4589)
![Screenshot 2024-09-23 233100](https://github.com/user-attachments/assets/0b5c8a03-1ea5-4633-bcd0-ca4ada232cde)

Type of change

New feature (non-breaking change which adds functionality)

How should this be tested?

Navigate to the newly added 3D planet visualizer project in the repository.
Open the project in a web browser(Live server) and ensure that the visualizer loads without any errors.
Interact with the visualizer:
Rotate or zoom in/out on the planet to check for smooth rendering and transitions.
Click on different elements (if applicable) to verify interactivity.
Ensure that there are no console errors or warnings.
Check the responsiveness of the visualizer on various screen sizes to ensure it displays correctly on mobile, tablet, and desktop views.
Expected Outcome:
The 3D planet visualizer should render and function as expected, providing an interactive and visually appealing experience without any glitches or performance issues.



 Verify that the 3D planet visualizer renders correctly.
 Check for smooth interactions and animations.

Mandatory Tasks
 Make sure you have self-reviewed the code. A decent size PR without self-review might be rejected.
